src/main/java/com/dev/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,63 @@
+package com.dev.cinema.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table (name = "movieSession")
```

Comment:
```suggestion
@Table (name = "movie_session")
```
normally we are using underscore style for table names

Reply:
done